code/modules/cargo/console.dm

```diff
@@ -196,49 +179,48 @@
 				printed_beacons++//printed_beacons starts at 0, so the first one out will be called beacon # 1
 				beacon.name = "Supply Pod Beacon #[printed_beacons]"
 		if("add")
-			var/area/ship/current_area = get_area(src)
-			var/datum/supply_pack/pack = SSshuttle.supply_packs[text2path(params["id"])]
-			if( \
-				!pack || !charge_account?.has_money(pack.cost) || !istype(current_area) || \
-				!istype(current_area.mobile_port.current_ship.docked_to, /datum/overmap/outpost) \
-			)
-				return
+			if(istype(current_ship.docked_to, /datum/overmap/outpost))
+				var/datum/supply_pack/current_pack = locate(params["ref"])
```

Comment:
```suggestion
			var/datum/overmap/outpost/current_outpost = current_ship.docked_to
			if(istype(current_ship.docked_to))
				var/datum/supply_pack/current_pack = locate(params["ref"]) in current_outpost.supply_packs
```
safety measure? performance improvement? futureproofing? take your pick, this probably does at least one of those things